extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -129,27 +171,33 @@ export const AssetDetail = ({
     assetIssuer,
   });
 
-  if (!assetOperations && !isSorobanAsset) {
```

Comment:
I'm removing this `!isSorobanAsset` check. This was added a long time ago here: https://github.com/stellar/freighter/pull/738

This was supposed to be blocking showing history for custom tokens, but wasn't actually working